benchmarks/advection_reaction_3D/raja/CMakeLists.txt

```diff
@@ -105,6 +124,23 @@ if(BUILD_ARKODE AND BUILD_CVODE AND BUILD_IDA)
     install(TARGETS advection_reaction_3D_raja_mpicuda
       DESTINATION "${BENCHMARKS_INSTALL_PATH}/advection_reaction_3D/raja")
 
+    set(BENCHMARK_VAR
+        "--method ARK-IMEX --nls tl-newton --tf 0.01 --dont-save\;arkimex_tlnewton_raja_mpicuda"
+        "--method ARK-DIRK --nls newton --tf 0.01 --dont-save\;arkdirk_newton_raja_mpicuda"
+        "--method CV-BDF --nls newton --tf 0.01 --dont-save\;cvbdf_newton_raja_mpicuda"
+        "--method IDA --nls newton --tf 0.01 --dont-save\;ida_newton_raja_mpicuda")
+
+    # NUM_CORES based on Lassen system for 1 node
+    foreach(benchmark_tuple ${BENCHMARK_VAR})
+      list(GET benchmark_tuple 0 benchmark_args)
+      list(GET benchmark_tuple 1 identifier)
+
+      sundials_add_benchmark(advection_reaction_3D_raja_mpicuda advection_reaction_3D_raja_mpicuda advection_reaction_3D
+        ENABLE_GPU
+        NUM_CORES 4
```

Comment:
Add a CMake option that is SUNDIALS_BENCHMARKS_NUM_GPUS and use it instead of `4` (default value can be 4).